op-guide/tidb-v3.0-upgrade-guide.md

```diff
@@ -0,0 +1,127 @@
+---
+title: TiDB 3.0 升级操作指南
+category: deployment
+---
+
+# TiDB 3.0 升级操作指南
+
+本文档适用于从 TiDB 2.0 版本（v2.0.1 及之后版本）或 TiDB 2.1 RC 版本升级到 TiDB 3.0 GA 版本。TiDB 3.0 版本兼容 [Kafka 版本的 TiDB-Binlog](/tools/tidb-binlog-kafka.md) 以及[集群模式的 TiDB-Binlog](/tools/tidb-binlog-cluster.md)。
+
+## 升级兼容性说明
+
+- loading
+
+## 注意事项
+
+- loading
+
+## 在中控机器上安装 Ansible 及其依赖
+
+TiDB-Ansible release-3.0 版本依赖 2.5.14 Ansible 版本（`ansible=2.5.14`），另依赖 Python 模块：`jinja2>=2.9.6` 和 `jmespath>=0.9.0`。为方便管理依赖，新版本使用 `pip` 安装 Ansible 及其依赖，可参照[在中控机器上安装 Ansible 及其依赖](../op-guide/ansible-deployment.md#在中控机器上安装-ansible-及其依赖) 进行安装。离线环境参照[在中控机器上离线安装 Ansible 及其依赖](../op-guide/offline-ansible-deployment.md#在中控机器上离线安装-ansible-及其依赖)。
+
+安装完成后，可通过以下命令查看版本：
+
+```
+$ ansible --version
+ansible 2.5.14
+$ pip show jinja2
+Name: Jinja2
+Version: 2.10
+$ pip show jmespath
+Name: jmespath
+Version: 0.9.0
+```
+
+> **注意：**
+>
+> 请务必按以上文档安装 Ansible 及其依赖。确认 Jinja2 版本是否正确，否则启动 Grafana 时会报错。确认 jmespath 版本是否正确，否则滚动升级 TiKV 时会报错。
+
+## 在中控机器上下载 TiDB-Ansible
+
+以 `tidb` 用户登录中控机并进入 `/home/tidb` 目录，备份 TiDB 2.0 版本或 TiDB 2.1 版本的 tidb-ansible 文件夹：
+
+```
+$ mv tidb-ansible tidb-ansible-bak
+```
+
+下载最新 tidb-ansible release-2.1 分支，默认的文件夹名称为 `tidb-ansible`。
+
+```
+$ git clone -b release-2.1 https://github.com/pingcap/tidb-ansible.git
```

Comment:
```suggestion
$ git clone -b release-3.0 https://github.com/pingcap/tidb-ansible.git
```

Reply:
@liubo0127 PTAL

Reply:
Suggest change to 'git clone -b $tag https://github.com/pingcap/tidb-ansible.git'